src/test/regress/sql/upgrade_single_shard_table_after.sql

```diff
@@ -0,0 +1,20 @@
+-- check that we properly retained the single-shard table
+SELECT 1 FROM pg_dist_partition
+WHERE logicalrelid = 'citus_schema.null_shard_key'::regclass AND
+      partmethod = 'n' AND repmodel = 's' AND colocationid != 0;
+
+INSERT INTO citus_schema.null_shard_key (name) VALUES ('c');
+SELECT * FROM citus_schema.null_shard_key WHERE id <= 2 ORDER BY id;
+
+-- Tests in after schedule are run twice, so we would insert a row with id=3
+-- at the first run and a row with id=4 at the second run.
+SELECT COUNT(*) IN (1,2) FROM citus_schema.null_shard_key WHERE id IN (3,4);
```

Comment:
Instead of doing such a trick we usually do the thing we want to test in a transaction instead.
```suggestion
BEGIN;
INSERT INTO citus_schema.null_shard_key (name) VALUES ('c');
SELECT name FROM citus_schema.null_shard_key ORDER BY name;
ROLLBACK;
```

Reply:
Maybe I should not use a bigserial column then. Because the rollback doesn't have an effect on sequences.

Reply:
Like in my suggested change: If you don't show the `id` value in the output, I think it should be fine.

Reply:
Yeah it was also a nice option, but I don't think it's necessary to use a bigserial column anyway, so changed the tests as such.